src/Components/Home/Placeholders.js

```diff
@@ -8,7 +8,7 @@ function Placeholder({ list }) {
     <>
       <Skeleton shape="rectangle" height="3.2rem" className="p-mb-4" />
       <div className="p-d-flex p-flex-wrap p-jc-center">
-        {list.map((user, key) => {
+        {list.map((_, key) => {
```

Comment:
Why do you remove user here? Although not used, it makes it clear what in the array is being mapped over and might be used later if further features are added.